crates/re_types_core/src/size_bytes.rs

```diff
@@ -69,6 +69,17 @@ impl<T: SizeBytes> SizeBytes for Vec<T> {
     }
 }
 
+impl<T: SizeBytes> SizeBytes for VecDeque<T> {
+    /// Does not take capacity into account.
```

Comment:
why though? Shouldn't we? Not sure...

Reply:
Capacity can be pretty inconsistent, and consistency is the only thing we really care about for these stats at the moment because we work with percentages (see my comment in previous PR).